.github/workflows/desiger-check-quartz-update.yml

```
@@ -0,0 +1,64 @@
+      - 'backend/**'
+  workflow_dispatch:
+
+jobs:
+  check-quartz-update:
+    name: Check Quartz update
+    runs-on: ubuntu-latest
+    env:
+      tables-file: backend/src/Designer/Migrations/SqlScripts/QuartzTables/tables_postgres.sql
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)